x-pack/plugins/observability_solution/apm/common/correlations/types.ts

```diff
@@ -28,6 +28,7 @@ export interface ResponseHitSource {
 
 export interface ResponseHit {
   _source: ResponseHitSource;
```

Comment:
can we remove `_source` here entirely to make it really clear that it shouldn't be used? the same goes for the result we get from the APMEventClient. Can we disable `_source` by default and make it obvious in the type that `_source` will be undefined?

Reply:
We can use `_source: false` to disable source completely.

Reply:
Could doing this have negative impact for products outside of observability? Or do those products need to make this change too?

Reply:
I assume this is a type that is only used by the APM app. Similarly, the APM Event client is mostly only used in the APM app. Basically, my proposal is to make _source opt-in instead of the default, and have that reflected in our types.

Reply:
I couldn't find any usage outside of APM but I set `_source` as optional